app/scripts/metamask-controller.js

```diff
@@ -20,7 +20,6 @@ const MessageManager = require('./lib/message-manager')
 const PersonalMessageManager = require('./lib/personal-message-manager')
 const TransactionController = require('./controllers/transactions')
 const ConfigManager = require('./lib/config-manager')
-const autoFaucet = require('./lib/auto-faucet')
```

Comment:
If we're removing a dependency, we should remove the old file, too, to keep the project folder clean.